app/services/idv/steps/verify_base_step.rb

```diff
@@ -49,10 +49,11 @@ def add_proofing_costs(results)
           if stage == :resolution
             # transaction_id comes from ConversationId
             add_cost(:lexis_nexis_resolution, transaction_id: hash[:transaction_id])
-            tmx_id = hash[:threatmetrix_request_id]
-            add_cost(:threatmetrix, transaction_id: tmx_id) if tmx_id
           elsif stage == :state_id
             process_aamva(hash[:transaction_id])
+          elsif stage == :threatmetrix
+            tmx_id = hash[:transaction_id]
+            add_cost(:threatmetrix, transaction_id: tmx_id) if tmx_id
```

Comment:
was this not caught as part of earlier refactor? can we add a regression spec that would have caught this?

Reply:
yeah this was a bug. it was adding costing for tmx at the resolution step and checking for proofingcomponent but getting the transaction_id from resolution not tmx.  i'll update the spec

Reply:
Any update on this? haven't seen specs for it yet